src/DIRAC/DataManagementSystem/Agent/RequestOperations/RegisterFile.py

```diff
@@ -95,7 +95,14 @@ def __call__(self):
             # # get LFN
             lfn = opFile.LFN
             # # and others
-            fileTuple = (lfn, opFile.PFN, opFile.Size, self.operation.targetSEList[0], opFile.GUID, opFile.Checksum)
+
+            # CHRIS 22.03.14
```

Comment:
We have he git log, so we don't need the date.

Reply:
done